src/cachingreader.cpp

```diff
@@ -11,6 +11,20 @@
 #include "util/math.h"
 #include "util/assert.h"
 
+namespace {
+
+// To prevent every bit of code having to guess how many samples
+// forward it makes sense to keep in memory, the hinter can provide
+// either 0 for a forward hint or -1 for a backward hint. We should
+// be calculating an appropriate number of samples to go backward as
+// some function of the latency, but for now just leave this as a
```

Comment:
some time ago we have changed the wording from "latency" to "audio buffer". 
Did you mean here: "as some function of the selected audio buffer in hardware preferences?"